clients/client-python/gravitino/api/credential/adls_token_credential.py

```diff
@@ -27,12 +27,12 @@ class ADLSTokenCredential(Credential, ABC):
 
     ADLS_TOKEN_CREDENTIAL_TYPE: str = "adls-token"
     ADLS_DOMAIN: str = "dfs.core.windows.net"
-    _STORAGE_ACCOUNT_NAME: str = "azure-storage-account-name"
-    _SAS_TOKEN: str = "adls-sas-token"
+    STORAGE_ACCOUNT_NAME: str = "azure-storage-account-name"
```

Comment:
please don't use the internal `STORAGE_ACCOUNT_NAME`  properties directly.  seem to other credentials.

Reply:
`seem to other credentials.`, What's the meaning of this sentences?

Reply:
it shoud be `same`

Reply:
How can I get constant value?  use method `getXXXX`or just create a new constants with the same value?